seed/python-sdk/file-upload/src/seed/resources/service/client.py

```diff
@@ -46,9 +49,13 @@ def post(
 
             - integer: int.
 
-            - file: typing.IO.
+            - file: FileTypes.
```

Comment:
I think it would be nice to append this string `See core.FileTypes for more documentation` to any file upload property. Ideally the `core.FileTypes` should link to the class or type directly.

Reply:
Updated, though it doesn't match the rest of the usage of classes from `core`